### DISCUSSION TITLE
Dotplot with selected cluster on Y axis

### DISCUSSION THREAD

**Discussion by singcell** [satijalab/seurat#6198] [Q&A] [answered]
When I use Dotplot function of Seurat then it displays all the UMAP cell clusters on Y axis. Is it possible to customize the graph and include only selected cluster or a clusters on Y axis?

**Comment by samuel-marsh** [comment, marked as answer]
Hi,

See `idents` parameter.  https://satijalab.org/seurat/reference/dotplot

Best,
Sam

**Reply by singcell** [reply]
Thank you!
